book/src/standard-types-and-functions.md

```diff
@@ -17,6 +17,53 @@ When compiling an expression, the compiler automatically inserts a small prelude
 
 Gluon has support for cooperative threading and communication between them through the `Thread` and `Sender`/`Receiver` types.
 
+### Strings
+
+`String` is a built-in data type. The module `std.string` provides the infix operatoin `++`, concatenating two strings. The operation `show` converts the Int to a printable String.
+```
+let io @ { ? } = import! std.io
+let string @ { (++), len } = import! std.string
```

Comment:
`++` is already in the prelude so you do not need to import it again https://github.com/gluon-lang/gluon/blob/25d949b8836d0cd28adc027f8d0e4e20a102aca1/src/lib.rs#L875

Reply:
done